### ISSUE TITLE
Add Token -- Cannot add legit token "0x59adcf176ed2f6788a41b8ea4c4904518e62b6a4"

### ISSUE THREAD

**Issue by chikeichan** [MetaMask/metamask-extension#2407]

**Comment by chikeichan** [comment]
@danfinlay 

I am getting the following error when calling `contract.at('0x59adcf176ed2f6788a41b8ea4c4904518e62b6a4')`. Any idea why?

```
index.js:76 Uncaught (in promise) Error: [ethjs-contract] while formatting incoming raw call data "0x5341490000000000000000000000000000000000000000000000000000000000" Error: Number can only safely store up to 53 bits
    at newMethodCallback (index.js:76)
    at cb (index.js:65)
    at index.js:58
    at index.js:68
```

**Comment by danfinlay** [comment]
Fun problem.

I tried adding the token in metamask, instantly got the same result. 
I put a breakpoint at the line of code where this was an issue, it was a problem with the return value from the contract, which implies a mismatch between the token ABI (ERC-20 standard) and the data type this contract returned.
I opened the network tab of the background process and inspected the request that returned this result.
The data field of the outgoing request was `0x95d89b41`.
I entered that 4-byte key into [the 4byte.directory](https://www.4byte.directory/signatures/?bytes4_signature=0x95d89b41) to see the method being called was `symbol()`.
Looking at [the contract on EtherScan](https://etherscan.io/token/0x59adcf176ed2f6788a41b8ea4c4904518e62b6a4#readContract), I see the `symbol()` function in this case returns a `bytes32` data type.
Looking at [the abi we use](https://github.com/danfinlay/human-standard-token-abi/blob/master/index.js), I can see we expect a `string` back.

So in conclusion:
- This contract doesn't adhere to the `human-standard-token` as we respect it.
- While we respect contracts that don't define a `symbol()` (it's not part of ERC-20, it's the expanded human-standard), we don't apparently have resilience for tokens that implement the standard methods *differently*.

Solutions:
- MetaMask could add some resilience for this kind of thing, could even fallback check other types in case a token uses a weird ABI?
- That token's maintainer should start returning `string` for `symbol()`.
